sample/CSharp/HttpTrigger-Cancellation/run.csx

```diff
@@ -5,22 +5,8 @@ using Microsoft.Extensions.Primitives;
 
 public static async Task<IActionResult> Run(HttpRequest req, ILogger log, CancellationToken token)
```

Comment:
Do we even need a cancellation sample if we've gotten rid of the cancellation token?

Reply:
In a way this test is not needed as it is testing for something that should not happen. However, since we are explicitly removing the cancellation token, this helps validate that change.

Reply:
Then it would make sense to keep the sample code the same, but change the assert to check that it's OK, right?

Reply:
We can certainly do that. The only reason i changed the sample code is because the original function code seems to indicate that it expects function OperationCanceledException. Somone reading the test / sample code without context would be confused as why it is there in the first place.

Reply:
Have added a tracking item for test changes.  https://github.com/Azure/azure-functions-host/issues/8084.

Reply:
@liliankasem flagging this as resolved as @soninaren will continue to iterate and follow up on your concerns using the issue he created.